### PR TITLE
'Fixed typos and grammar errors'

### PR DESCRIPTION
**Pull request by saulecabrera**
@igas,
I fixed some typos in the README as well as some grammar errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ If you want to use `faker` outside tests remove `, only: :test` part.
...
 ## Usage
 
-You nead to start `:faker` application. Cause of many usages of fake data can be
-(seed database, tests, etc) it's no right place to start it. For example, if you
+You need to start `:faker` application, but due to the many usages of fake data,
+(seed database, tests, etc) there's no right place to do it. For example, if you
 want to use it in tests, just add `Faker.start` to `test/test_helper.exs`, and
-use any function decribed below. For example, if you nead random name use
-`Faker.Name.name`.
+use any function decribed below. For example, if you need a random name use
```
Comment (igas) [inline README.md]: `described below` doesn't make any sense now. We should point to http://hexdocs.pm/faker

**Comment by saulecabrera** [comment]
Hey @igas,

I added the link to the documentation. Check it out to see if it fits :)

**Comment by igas** [comment]
Thanks for great work!

**Comment by saulecabrera** [comment]
No problem! @igas

**Pull request merged**